scripts/tests/perf/relayed-udp-server2client.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -euox pipefail
+
+source "./scripts/tests/perf/force-relayed.sh"
```

Comment:
You could reuse the existing "lib" here I think. Also sending the ping is the same, right?

Reply:
Good call. I removed the ping -- no longer necessary.